tests/verific/clocking.ys

```diff
@@ -0,0 +1,10 @@
+read -sv <<EOT
+module test(input clk, foo);
```

Comment:
Oh, the `clk` input is unnecessary for this test, it was just there for debugging so I could switch quickly between `always @(*)` and `always @(posedge clk)`

Reply:
Updated